website/docs/user-group-role/access-control/manage_permissions.md

```diff
@@ -0,0 +1,95 @@
+---
+title: "Manage permissions"
+description: "Learn how to use global and object permissions in authentik."
+---
+
+Refer to the following topics for instructions to view and manage permissions.
+
+## View permissions
+
+You can view all permissions that are assigned to a user, group, or role.
+
+### View user permissions
+
+To view _object_ permissions for a specific user:
+
+1. Go to the Admin interface and navigate to **Directory -> Users**.
+2. Select a specific user by clicking on the name, to open the user's details page.
+3. Click the **Permissions** tab at the top of the page.
+
+To view _global_ permissions for a user:
+
+1. Go to the Admin interface and navigate to **Directory -> Users**.
+2. Select a specific user by clicking on the name, to open the user's details page.
+3. Click the **Assigned Permissions** tab at the top of the page (to the right of the **Permissions** tab).
+
+### View group permissions
```

Comment:
Putting groups here makes it confusing since permissions are not assigned directly to groups

Reply:
Ah true.. there are user-object-permissions and role-object-permissions that are assigned TO users and roles for that they can do TO the group.. but the group cant do anything itself.

I think this is now fixed by how I restructured the docs, but we can tweak more as needed.